Added calls host control functionality to docs (#7204)
mattermost/docs: source/collaborate/make-calls.rst

  On iOS, audio will automatically come through a connected device. You can override this behavior by tapping the **Speaker** button. Audio will then come through the speaker. However, you cannot manually select an output device on iOS at this time.

Host controls
-------------

From Mattermost v9.9, call host controls are available when using Mattermost in a web browser, the desktop app, or the mobile apps, and include the ability to `transfer host duties <#transfer-host-duties>`__, `remove call participants <#remove-call-participants>`__, `stop a screen share <#stop-a-screen-share>`__, `mute or unmute participants <#mute-or-nmute-participants>`__, and `lower raised hands <#lower-raised-hands>`__. 

Access host controls
~~~~~~~~~~~~~~~~~~~~

> Host controls are available to call hosts in both the call widget by selecting the **More** |more-icon| icon next to a participant's name, and in the expanded the call window as hosts hover over a call participant in the list.

Reviewer: Maybe:
```suggestion
Host controls are available to call hosts and admins in both the call widget by selecting the **More** |more-icon| icon next to a participant's name, and in the expanded the call window as hosts hover over a call participant in the list.
```

Transfer host duties
~~~~~~~~~~~~~~~~~~~~

> Transfer host duties to another call particpant by accessing the host controls and selecting **Make host**. Once host duties are transfered to someone else, you can't access host controls unless they're transferred back to you.

Reviewer: ```suggestion
Transfer host duties to another call participant by accessing the host controls and selecting **Make host**. Once host duties are transferred to someone else, you can't access host controls unless they're transferred back to you. System admins can change the host at any time.
```

Remove call participants
~~~~~~~~~~~~~~~~~~~~~~~~

Remove a call participant from an active call by accessing the host controls and selecting **Remove from call**, then confirm by selecting **Yes, remove**. The call participant is notified that they've been removed from the call by the host.

Stop a screen share
~~~~~~~~~~~~~~~~~~~~

Stop a call participant's screen share by accessing the host controls and selecting **Stop screen share**.

> Mute or unmute participants

Reviewer: We didn't implement the unmute participants feature, so:
```suggestion
Mute participants
```

Reviewer: And I suppose the below needs to be changed too :)

~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Invite muted participants to unmute their microphone by accessing the host controls and selecting **Ask to unmute**.  The call particpant is prompted to decide whether unmute or stay muted.

You can mute or unmute the microphone of specific particpants by accessing the host controls and toggling the particpant's mute icon. Mute all call particpants by selecting **Mute all**, and unmute all participants by selecting **Unmute all**.

Lower raised hands
~~~~~~~~~~~~~~~~~~~

> Lower a raised hand by accessing the host controls and selecting **Lower hand**.

Reviewer: ```suggestion
Lower a raised hand by accessing the host controls and selecting **Lower hand**. The participant will be notified that their hand was lowered by the host.
```

Share your screen
-----------------

> During a call, all call participants can share your screen with other call participants, unless your system admin has :ref:`disabled your ability to do so <configure/plugins-configuration-settings:allow screen sharing>`. 

Reviewer: Hm, I think the original makes more sense, imo. We don't allow participants or hosts to share other people's screens. Unless you meant `all call participants can share their screen`?

Author: Great flag! I've corrected "your screen" to "their screen" for clarity.
